hack/generate-crd-yaml.sh

```diff
@@ -16,6 +16,6 @@ cd $REPO
 # so, output them to separate files, then concatenate those files.
 # That should give a stable sort.
 go run sigs.k8s.io/controller-tools/cmd/controller-gen \
-  crd paths=./apis/... output:dir=$TEMPDIR
+  crd paths=./apis/projectcontour/... output:dir=$TEMPDIR
```

Comment:
Same as above ... if it's not necessary, we should leave this as is.

Reply:
This one ensures that the example install no longer includes the `contour.heptio.com` CRDs.

Reply:
I will make sure that I update it back when we do remove the actual CRDs.